gobblin-data-management/src/main/java/gobblin/data/management/conversion/hive/dataset/ConvertibleHiveDataset.java

```diff
@@ -136,6 +136,7 @@ public ConvertibleHiveDataset(FileSystem fs, HiveMetastoreClientPool clientPool,
     public static final String DESTINATION_TABLE_KEY = "destination.tableName";
     public static final String DESTINATION_DB_KEY = "destination.dbName";
     public static final String DESTINATION_DATA_PATH_KEY = "destination.dataPath";
+    public static final String DESTINATION_TABLE_COMPRESSION_KEY = "destination.tableCompression";
```

Comment:
Do you think the property should be ```destination.additionalTableProperties``` instead of ```destination.tableCompression```? This will allow us to add more table properties without any code change.